python/pyspark/ml/feature.py

```diff
@@ -57,8 +58,8 @@ class Binarizer(JavaTransformer, HasInputCol, HasOutputCol):
     .. versionadded:: 1.4.0
     """
 
-    threshold = Param(Params._dummy(), "threshold",
-                      "threshold in binary classification prediction, in range [0, 1]")
+    threshold = FloatParam(Params._dummy(), "threshold",
```

Comment:
You're correct, I didn't catch that. I have updated it to the correct description.